clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -1122,10 +1108,10 @@ void handleFailure(Throwable throwable) {
 
     @Override
     public ListTopicsResult listTopics(final ListTopicsOptions options) {
-        final KafkaFutureImpl<Map<String, TopicListing>> topicListingFuture = new KafkaFutureImpl<>();
+        final KafkaFutureImpl<Map<String, TopicListItem>> topicListingFuture = new KafkaFutureImpl<>();
```

Comment:
OK, I will revert. It's a tough one because I would never guess what a `TopicListing` is from that name. I've never seen any API that uses the word `Listing` to refer to a more compact representation of an object. If I had to guess, I would have guessed that `listTopics` would return a `TopicListing` instead of a collection of `TopicListing`s. In other words, I'm more familiar with meaning `1` from the dictionary entry I quoted than meaning `1.2`.